src/nats.rs

```diff
@@ -14,6 +14,32 @@ use async_nats::{
 use tracing::{debug, warn};
 use wadm::DEFAULT_EXPIRY_TIME;
 
+#[derive(Debug, Clone, Copy, Default, clap::ValueEnum)]
+#[clap(rename_all = "PascalCase")]
+pub enum StreamPersistence {
+    #[default]
+    File,
+    Memory,
+}
```

Comment:
for these, what do you think about making these lowercase instead of pascal case? I know the NATS options are PascalCase, but it feels a little odd to me thinking about doing `wadm --stream-persistence File` over `wadm --stream-persistence file`